tools/AzureRM.BootStrapper/AzureRM.Bootstrapper.psm1

```diff
@@ -357,7 +359,23 @@ function Uninstall-ModuleHelper
         }
         catch
         {
-          Write-Error $_.Exception.Message
+          if ($_.Exception.Message -match "No match was found")
+          {
+            # Check for msi installation (Install folder: C:\ProgramFiles(x86)\Microsoft SDKs\Azure\PowerShell) Only in windows
+            if ((-not $Script:IsCoreEdition) -or ($IsWindows))
+            {
+              $sdkPath = (join-path ${env:ProgramFiles(x86)} -childpath "\Microsoft SDKs\Azure\PowerShell\")
```

Comment:
We need to check both ProgramFiles9x86) and ProgramFiles, as this could be a 32-bit system